packages/adapter-auto/package.json

```
@@ -13,7 +13,7 @@
 	],
 	"repository": {
 		"type": "git",
-		"url": "https://github.com/sveltejs/kit",
+		"url": "https://github.com/sveltejs/kit.git",
```

```suggestion
		"url": "git+https://github.com/sveltejs/kit.git",
```

These URLs may need a "git+" prefix too, according to publint:

```sh
~/git/sveltejs/kit-2/packages/adapter-auto pr/aewing/14134
❯ pnpx publint           
Packages: +6
++++++
Progress: resolved 6, reused 6, downloaded 0, added 6, done
Running publint v0.3.12 for @sveltejs/adapter-auto...
Packing files with `pnpm pack`...
Linting...
Suggestions:
1. pkg.repository.url is https://github.com/sveltejs/kit.git but could be a full git URL like "git+https://github.com/sveltejs/kit.git".
```